### ISSUE TITLE
[Question] How to properly move to 'SearchInputDecoration'

### ISSUE THREAD

**Issue by SonnyBlack** [maheshj01/searchfield#177]
Hi, I have been using your library (1.0.9). In this version, property 'searchInputDecoration' was as 'InputDecoration' class. Now I want to update your library and I need to change it to 'SearchInputDecoration'. I can't find 'prefixIconConstraints' in it, I use it because I need to adjust icon's size (it's too big and looks very ugly).  I see that 'SearchInputDecoration' extends from 'InputDecoration', but it seems like I can't to pass 'prefixIconConstraints' to it.
I'm not very familiar with flutter/dart, so could someone please explain to me how I can do this. Thanks in advance.

**Comment by maheshj01** [comment]
Hi @SonnyBlack, Thanks for filing the issue.
Looks like that properties is not added to `SearchInputDecoration`  I will update and publish a new release soon.

**Comment by maheshj01** [comment]
Added new properties and released in [v1.1.4](https://pub.dev/packages/searchfield/changelog)

cc: @SonnyBlack

**Comment by buxiaojiong** [comment]
"both prefixIcon and prefixIconConstraints can't be used at the same time" ? I want uses prefixIcon and prefixIconConstraints at the same time ,because in flutter Textfield can set.

**Comment by SonnyBlack** [comment]
Hi @maheshj01 . Thanks for the very quick reply! I have just tried v1.1.5. Now I am able to use `SearchInputDecoration` completely with all my settings that I passed to `InputDecoration`. But, I don't know why - it does not work. I use it in combination with 'prefixIcon'. I also did `flutter clean` - it does not help.

Here is an example of the current version in use (1.0.9)
<img width="401" alt="Screenshot 2024-09-26 at 12 11 09" src="https://github.com/user-attachments/assets/60e10d04-0df1-4423-871a-e46948cbf698"> 

And here is example of version (1.1.5)
<img width="406" alt="Screenshot 2024-09-26 at 12 03 08" src="https://github.com/user-attachments/assets/9cb69ab9-c639-4d3b-897b-3ccf5bea9adb">

The values of `prefixIconConstraints/prefixIcon` are the same
